src/nextjournal/clerk/hashing.clj

```diff
@@ -22,13 +22,15 @@
 (defn var-name
   "Takes a `form` and returns the name of the var, if it exists."
   [form]
-  (when (and (sequential? form)
+  (when (and (list? form)
```

Comment:
vectors are sequentials but you can't have `def`/`defn`s using vectors, so switching to `list?` is a bit tighter; added https://github.com/nextjournal/clerk/pull/26/files#diff-b33fbf28a0d44915162c46241d20d137f2ca22398ab87a4881637a3358159cc8R22 to check this

Reply:
I would not ever use `list?` but use `seq?` instead. `list?` will return false for "lists" created by syntax-quote expressions which are typically constructed using `cons`.